CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
-cmake_minimum_required(VERSION 3.17)
+cmake_minimum_required(VERSION 3.24)
```

Comment:
needed for the behavior that I want in `FetchContent`, I'll explain on another comment

Reply:
3.24 is OK, it is quite old already.